codebase/superdarn/src.bin/tk/tool/make_grid.2.0/make_grid.c

```diff
@@ -681,29 +658,28 @@ int main(int argc,char *argv[]) {
             else stime+=sdate;
 
             /* If median filter is going to be applied then we need to load data
-             * prior to the usuals tart time, so stime needs to be adjusted */
+             * prior to the usuals start time, so stime needs to be adjusted */
             if (bxcar==1) {
                 /* subtract one src */
                 if (tlen !=0) stime-=tlen;
                 else stime-=15+src[0]->ed_time-src[0]->st_time; /* pad to make sure */
             }
 
             /* Calculate year, month, day, hour, minute, and second of 
-             * grid start time */
+             * grid start time (needed to load AACGM_v2 coefficients) */
             TimeEpochToYMDHMS(stime,&yr,&mo,&dy,&hr,&mt,&sc);
-
             /* Search for index of corresponding record in input file given
              * grid start time */
             if (fitflg) {
                 /* Input file is in fit or fitacf format */
 
                 if (old) s=OldFitSeek(oldfitfp,yr,mo,dy,hr,mt,sc,NULL);
                 else s=FitFseek(fitfp,yr,mo,dy,hr,mt,sc,NULL,inx);
-
-                /* If a matching record could not be found then exit */
+                /* If a matching record could not be found then exit*/ 
                 if (s ==-1) {
-                    fprintf(stderr,"File does not contain the requested interval.\n");
-                    exit(-1);
+                    fprintf(stderr,"File does not contain the requested start time, ignoring this file.\n");
```

Comment:
Responding to @billetd and @mts299 - is this clearer?
```suggestion
                    fprintf(stderr,"File ends before requested start time, ignoring this file.\n");
```